swagger/v1/api_spec.json

```diff
@@ -1466,8 +1482,7 @@
           "participant_type",
           "cohort",
           "status",
-          "training_status",
-          "updated_at"
+          "training_status"
```

Comment:
I'm not sure why this has changed. I ran `bundle exec rake rswag:specs:swaggerize` to generate this.